pyeudiw/trust/handler/direct_trust_sd_jwt_vc.py

```diff
@@ -0,0 +1,129 @@
+import os
+from pyeudiw.trust.handler.interface import TrustHandlerInterface
+from pyeudiw.trust.model.trust_source import TrustSourceData
+from pyeudiw.tools.base_logger import BaseLogger
+from pyeudiw.tools.utils import get_http_url
+from urllib.parse import ParseResult, urlparse
+from typing import Literal
+from pyeudiw.tools.utils import cacheable_get_http_url, get_http_url
+from pyeudiw.trust.handler.exception import InvalidJwkMetadataException
+
+
+DEAFAULT_JWK_ENDPOINT = "/.well-known/jwt-vc-issuer"
+DEAFAULT_METADATA_ENDPOINT = "/.well-known/openid-credential-issuer"
+
+DEFAULT_DIRECT_TRUST_SD_JWC_VC_PARAMS = {
+    "connection": {
+        "ssl": os.getenv("PYEUDIW_HTTPC_SSL", True)
+    },
+    "session": {
+        "timeout": os.getenv("PYEUDIW_HTTPC_TIMEOUT", 6)
+    }
+}
+
+class DirectTrustSdJwtVc(TrustHandlerInterface, BaseLogger):
```

Comment:
The class `DirectTrustSdJwtVc` does not satisfy the `TrustHandlerInterface` interface as it does not implement the method `verify()`

Reply:
Resolved in [33c0a54](https://github.com/italia/eudi-wallet-it-python/pull/284/commits/33c0a54145e85bc387e4b4420245fc4af56abac8)